src/github.com/getlantern/cloudflare/LICENSE

```diff
@@ -0,0 +1,362 @@
+Mozilla Public License, version 2.0
```

Comment:
I added the CloudFlare package to flashlight-build.  There are no changes to it.